app/Main.hs

```diff
@@ -3,7 +3,8 @@ module Main where
 import Data.Char
 import SIL
 --import SIL.Llvm
-import SIL.Parser
+import SIL.ParserMegaparsec
+-- import SIL.Parser
```

Comment:
Let's try to keep out commented-out code from commits. I have a bad habit of it myself.